Error with leader election examples

Simple java integration:


```java
import org.apache.camel.builder.RouteBuilder;

public class Master extends RouteBuilder {
  @Override
  public void configure() throws Exception {
	  from("master:lock:timer:tick")
	  .setHeader("m").constant("string!")
	  .setBody().simple("Magic${header.m}")
      .log("${body}");
  }
}

```

When running I get:

```
master-6f6d965bbb-vwvvg integration Exception in thread "main" java.lang.ExceptionInInitializerError
master-6f6d965bbb-vwvvg integration 	at io.quarkus.runner.ApplicationImpl.<clinit>(ApplicationImpl.zig:47)
master-6f6d965bbb-vwvvg integration 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
master-6f6d965bbb-vwvvg integration 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
master-6f6d965bbb-vwvvg integration 	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
master-6f6d965bbb-vwvvg integration 	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
master-6f6d965bbb-vwvvg integration 	at java.base/java.lang.Class.newInstance(Class.java:584)
master-6f6d965bbb-vwvvg integration 	at io.quarkus.runtime.Quarkus.run(Quarkus.java:61)
master-6f6d965bbb-vwvvg integration 	at io.quarkus.runtime.Quarkus.run(Quarkus.java:38)
master-6f6d965bbb-vwvvg integration 	at io.quarkus.runtime.Quarkus.run(Quarkus.java:104)
master-6f6d965bbb-vwvvg integration 	at io.quarkus.runner.GeneratedMain.main(GeneratedMain.zig:29)
master-6f6d965bbb-vwvvg integration Caused by: java.lang.RuntimeException: java.io.IOException: Is a directory
master-6f6d965bbb-vwvvg integration 	at org.apache.camel.k.support.RuntimeSupport.loadUserProperties(RuntimeSupport.java:375)
master-6f6d965bbb-vwvvg integration 	at org.apache.camel.k.quarkus.ApplicationConfigSourceProvider.getConfigSources(ApplicationConfigSourceProvider.java:31)
master-6f6d965bbb-vwvvg integration 	at io.smallrye.config.SmallRyeConfigBuilder.lambda$discoverSources$0(SmallRyeConfigBuilder.java:95)
master-6f6d965bbb-vwvvg integration 	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
master-6f6d965bbb-vwvvg integration 	at io.smallrye.config.SmallRyeConfigBuilder.discoverSources(SmallRyeConfigBuilder.java:95)
master-6f6d965bbb-vwvvg integration 	at io.smallrye.config.SmallRyeConfig.buildConfigSources(SmallRyeConfig.java:80)
master-6f6d965bbb-vwvvg integration 	at io.smallrye.config.SmallRyeConfig.<init>(SmallRyeConfig.java:63)
master-6f6d965bbb-vwvvg integration 	at io.smallrye.config.SmallRyeConfigBuilder.build(SmallRyeConfigBuilder.java:337)
master-6f6d965bbb-vwvvg integration 	at io.quarkus.runtime.generated.Config.<clinit>(Config.zig:293)
master-6f6d965bbb-vwvvg integration 	... 10 more
master-6f6d965bbb-vwvvg integration Caused by: java.io.IOException: Is a directory
master-6f6d965bbb-vwvvg integration 	at java.base/sun.nio.ch.FileDispatcherImpl.read0(Native Method)
master-6f6d965bbb-vwvvg integration 	at java.base/sun.nio.ch.FileDispatcherImpl.read(FileDispatcherImpl.java:48)
master-6f6d965bbb-vwvvg integration 	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
master-6f6d965bbb-vwvvg integration 	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:245)
master-6f6d965bbb-vwvvg integration 	at java.base/sun.nio.ch.FileChannelImpl.read(FileChannelImpl.java:223)
master-6f6d965bbb-vwvvg integration 	at java.base/sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:65)
master-6f6d965bbb-vwvvg integration 	at java.base/sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:109)
master-6f6d965bbb-vwvvg integration 	at java.base/sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:103)
master-6f6d965bbb-vwvvg integration 	at java.base/java.nio.file.Files.read(Files.java:3158)
master-6f6d965bbb-vwvvg integration 	at java.base/java.nio.file.Files.readAllBytes(Files.java:3212)
master-6f6d965bbb-vwvvg integration 	at java.base/java.nio.file.Files.readString(Files.java:3283)
master-6f6d965bbb-vwvvg integration 	at org.apache.camel.k.support.RuntimeSupport$1.visitFile(RuntimeSupport.java:362)
master-6f6d965bbb-vwvvg integration 	at org.apache.camel.k.support.RuntimeSupport$1.visitFile(RuntimeSupport.java:347)
master-6f6d965bbb-vwvvg integration 	at java.base/java.nio.file.Files.walkFileTree(Files.java:2724)
master-6f6d965bbb-vwvvg integration 	at java.base/java.nio.file.Files.walkFileTree(Files.java:2796)
master-6f6d965bbb-vwvvg integration 	at org.apache.camel.k.support.RuntimeSupport.loadUserProperties(RuntimeSupport.java:373)
master-6f6d965bbb-vwvvg integration 	... 18 more
```

--- a/camel-k-core/support/src/main/java/org/apache/camel/k/support/RuntimeSupport.java
+++ b/camel-k-core/support/src/main/java/org/apache/camel/k/support/RuntimeSupport.java
@@ -322,7 +322,7 @@ public final class RuntimeSupport {
         try {
             Path confPath = Paths.get(conf);
 
-            if (Files.exists(confPath)) {
+            if (Files.exists(confPath) && !Files.isDirectory(confPath)) {
                 try (Reader reader = Files.newBufferedReader(confPath)) {
                     Properties p = new Properties();
                     p.load(reader);
@@ -349,6 +349,10 @@ public final class RuntimeSupport {
             public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IOException {
                 Objects.requireNonNull(file);
                 Objects.requireNonNull(attrs);
+
+                if (Files.isDirectory(file)) {
+                    return FileVisitResult.CONTINUE;
+                }
 
                 if (file.toFile().getAbsolutePath().endsWith(".properties")) {
                     try (Reader reader = Files.newBufferedReader(file)) {

